UM/Version.py

```diff
@@ -14,7 +15,7 @@ class Version:
     #   the major, minor and revision version numbers. All text is ignored.
     #
     #   \param version A string or bytes representing a version number.
-    def __init__(self, version: Union[str, bytes, List[Union[int, str]]]) -> None:
+    def __init__(self, version: Union[str, bytes, List[Union[int, str]], int]) -> None:
```

Comment:
It might be a bit more readable if int was together with str and bytes instead of behind the List thing.